### PR TITLE
cleanup feature code after activated everywhere

### PR DESCRIPTION
**Pull request by taozhu-chicago**
#### Problem

Feature `A8xyMHZovGXFkorFqEmVH2PKGLiBip5JD7jt4zsUWo4H` Remove congestion multiplier from transaction fee calculation #29881 has been activated everywhere.

#### Summary of Changes
- clean up feature code

Fixes #

### CODE REVIEW

File: sdk/src/fee.rs [hunk 1]
```diff
@@ -82,18 +82,13 @@ impl FeeStructure {
         message: &SanitizedMessage,
         lamports_per_signature: u64,
         budget_limits: &FeeBudgetLimits,
-        remove_congestion_multiplier: bool,
         include_loaded_account_data_size_in_fee: bool,
     ) -> u64 {
         // Fee based on compute units and signatures
         let congestion_multiplier = if lamports_per_signature == 0 {
```
Comment (apfitzge) [inline sdk/src/fee.rs:88]: which tests are affected by this remaining branch? Seems unecessary to have this branch with the concept of congestion multiplier removed.

**Review by apfitzge — APPROVED** [review]
lgtm - the mixed ordering of `remove_congestion_multiplier` and `include_loaded_account_data_size_in_fee` in calculate_fee and calculate_test_fee made this confusing to review!

**Comment by taozhu-chicago** [comment]
> lgtm - the mixed ordering of `remove_congestion_multiplier` and `include_loaded_account_data_size_in_fee` in calculate_fee and calculate_test_fee made this confusing to review!

good news is it's better now :D

File: sdk/src/fee.rs [hunk 2]
```diff
@@ -82,18 +82,13 @@ impl FeeStructure {
...
             0.0 // test only
-        } else if remove_congestion_multiplier {
-            1.0 // multiplier that has no effect
         } else {
-            const BASE_CONGESTION: f64 = 5_000.0;
-            let current_congestion = BASE_CONGESTION.max(lamports_per_signature as f64);
-            BASE_CONGESTION / current_congestion
+            1.0 // multiplier that has no effect
```
Comment (steviez) [inline sdk/src/fee.rs:91]: If the multiplier is just 1.0, rip out the variable altogether? One less float to throw into the multiply at the bottom of the function

Comment (taozhu-chicago) [reply sdk/src/fee.rs:91]: Yeah, working on a follow up PR to remove the "for test" case all together

**Comment by steviez** [comment]
Was a tad late to the review, but looks good to me; sounds like Tao will address my last comment in a further comment:
```
$ solana feature status | grep A8xyMHZovGXFkorFqEmVH2PKGLiBip5JD7jt4zsUWo4H
A8xyMHZovGXFkorFqEmVH2PKGLiBip5JD7jt4zsUWo4H | active since epoch 548  | 236736000       | Remove congestion multiplier from transaction fee calculation #29881
```

**Comment by taozhu-chicago** [comment]
> Was a tad late to the review, but looks good to me; sounds like Tao will address my last comment in a further comment:
> 
> ```
> $ solana feature status | grep A8xyMHZovGXFkorFqEmVH2PKGLiBip5JD7jt4zsUWo4H
> A8xyMHZovGXFkorFqEmVH2PKGLiBip5JD7jt4zsUWo4H | active since epoch 548  | 236736000       | Remove congestion multiplier from transaction fee calculation #29881
> ```

#34537 is the follow up PR to remove unnecessary parameter, leads to a great simplification of code

**Pull request merged**